packages/teams-js/test/private/logs.spec.ts

```diff
@@ -29,6 +29,13 @@ describe('logs', () => {
     }
   });
 
+  describe('Testings logs.isSupported', () => {
+    it('should throw if called before initialization', () => {
+      utils.setRuntimeConfig(_uninitializedRuntime);
+      expect(() => logs.isSupported()).toThrowError('The library has not yet been initialized');
+    });
+  });
+
   describe('Testing logs.registerGetLogHandler function', () => {
     it('logs.registerGetLogHandler should not allow calls before initialization', () => {
```

Comment:
>logs.registerGetLogHandler should not allow calls before initialization [](http://example.com/codeflow?start=8&length=71)

Should there be a test case that validates calling `registerGetLogHandler` before initialize when not passing a handler does NOT throw?

Reply:
Same question for the three other ones in `pages` that also intentionally don't do stuff when no `handler` is passed.

Reply:
Probably

Reply:
Done

Reply:
I think maybe my suggestion wasn't clear -- from my reading of the `registerGetLogHandler` code, it intentionally does NOT throw, even if `initialize` hasn't finished, if no `handler` was passed.  Is there a unit test case for that? I see a test case here for ensuring that it throws when calling `registerGetLogHandler` when given a handler has a parameter, but I'm talking about the different behavior when no handler parameter is passed.